web/config.py

```diff
@@ -192,6 +192,9 @@
 # Hashing algorithm used for password storage
 SECURITY_PASSWORD_HASH = 'pbkdf2_sha512'
 
+# Minimum password length
+SECURITY_PASSWORD_LENGTH_MIN = 6
```

Comment:
Will this apply when adding new user from users dialog?

Reply:
users dialog is already having validation on 6 characters.

Reply:
OK.